packages/terra-toolbar/src/Toolbar.jsx

```diff
@@ -12,6 +12,16 @@ const propTypes = {
    * Content alignment. Align to start, end, or center.
    */
   align: PropTypes.oneOf(['start', 'end', 'center']),
+  /**
+   * String that labels the content the toolbar controls.
+   */
+  ariaLabel: PropTypes.string,
+  /**
+   * String that labels the content the toolbar controls. References the
+   * id of the element the toolbar controls. Will be used over ariaLabel if
+   * both are present.
+   */
+  ariaLabelledBy: PropTypes.string,
```

Comment:
> References the id of the element the toolbar controls.

This is not accurate. It's not the ID of the element that the toolbar controls. The ID provided to `ariaLabelledBy` should be the visible text label -- usually is a label div, span, heading, etc. -- that describes what the toolbar does (e.g. "Text Formatting"). If no visible label is present, then `ariaLabel ` should be used instead.


Another attribute + prop that we should add to appropriately add programmatic context for what element the toolbar does control would be `'aria-controls'`:
https://www.w3.org/WAI/ARIA/apg/example-index/toolbar/toolbar.html